src/main.scss

```diff
@@ -0,0 +1,801 @@
+/* Fonts */
+@font-face {
+  font-family: "et-line";
+  src: url("../resources/et-line-font/fonts/et-line.woff") format("woff"),
+    url("../resources/et-line-font/fonts/et-line.ttf") format("truetype"),
+    url("../resources/et-line-font/fonts/et-line.svg#et-line") format("svg");
+}
+
+/* Changing bootstrap default grid breakpoints */
+$grid-breakpoints: (
+  xs: 0,
+  sm: 577px,
+  md: 769px,
+  lg: 992px,
+  xl: 1200px,
+);
+
+@import "../node_modules/bootstrap/scss/bootstrap";
+
+/* Global */
+
+* {
+  font-family: "Oxygen", sans-serif;
+  box-sizing: border-box;
+}
+html,
+body {
+  max-width: 100%;
+  overflow-x: hidden;
+  scroll-behavior: smooth;
+  height: 100%;
+}
+
+p {
+  color: gray;
+}
+
+h1 {
+  font-weight: bold;
+}
+
+h4 {
+  font-weight: 700;
+}
+
+.fs1 {
+  font-size: 2.5rem;
+}
+
+.custom-icon {
+  background-color: black;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 10px;
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
+  border-radius: 50%;
+  margin-bottom: 0.5rem;
+  .icon {
+    display: inline-block;
+    width: 1em;
+    height: 1em;
+    stroke-width: 0;
+    stroke: currentColor;
+    fill: currentColor;
+    background-color: #000;
+    color: white;
+    border-radius: 20%;
+  }
+}
+
+/* 
+0. Colors
+1. Main banner
+2. Nav Bar
+3. services
+4. about
+5. team
+6. portfolio
+7. blog
+8. questions
+9. testimonials
+9. contact
+10. footer
+*/
+
+/* COLORS */
+$main-color: #64c6ae;
+/* END COLORS */
+
+/* MAIN BANNER */
+#main-banner {
```

Comment:
don't use ids for style, unless you want to use it as specificity to override a style